docker/shadow-hand/indigo/nvidia_ide/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
here you have description in Dockerfile. Please remove this comment since it doesn't contain any useful information